src/applications/find-forms/components/SearchResult.jsx

```diff
@@ -70,6 +70,38 @@ const recordGAEventHelper = ({
     'search-typeahead-enabled': false, // consistently populate with false since there's no type ahead enabled for this search feature
   });
 
+const deriveRelatedTo = ({
+  vaFormAdministration,
+  formType,
+  benefitCategories,
+}) => {
+  let relatedTo = vaFormAdministration;
+
+  if (formType === 'employment') {
+    relatedTo = 'Employment or jobs at VA';
+  }
+  if (formType === 'non-va') {
+    relatedTo = 'A non-VA form. For other government agency forms, go to the ';
+  }
+
+  if (benefitCategories?.length > 0) {
+    relatedTo = benefitCategories.map(f => f.name).join(', ');
+  }
+
+  if (relatedTo) {
+    return (
+      <dd className="vads-u-margin-y--1 vads-u-margin-y--1">
+        <dfn className="vads-u-font-weight--bold">Related to:</dfn> {relatedTo}
+        {formType === 'non-va' ? (
+          <a href="https://www.gsa.gov/reference/forms">GSA forms library</a>
+        ) : (
+          ''
+        )}
+      </dd>
+    );
+  } else return null;
```

Comment:
Nit - you don't need the `else` here since there is a `return` inside the `if`. The last line can just be `return null`.